pkg/processor/runtime/python/py/_nuclio_wrapper.py

```diff
@@ -235,7 +235,7 @@ def _load_entrypoint_from_handler(self, handler):
         try:
             entrypoint_address = getattr(module, entrypoint)
         except Exception as err:
-            self._logger.warn_with('Handler not found', handler=handler)
+            self._logger.error_with('Handler not found', handler=handler)
```

Comment:
imo there's nothing informative about logging `handler` because it is a bounded function,
maybe log the module name? the entrypoint?

Reply:
there's every information needed in `handler`. it is composed of `module:entrypoint` in the way the user passes it as input

Reply:
Im saying that because handler is a function, and once printed, it would look like `<function handler at 0xXXXXX>`

Reply:
`handler` is a string. `entrypoint_address`is the function

Reply:
alright then!